server/mock/mock_relay.go

```diff
@@ -153,9 +155,9 @@ func (m *mockRelay) handleRegisterValidator(w http.ResponseWriter, req *http.Req
 }
 
 // defaultHandleRegisterValidator returns the default handler for handleRegisterValidator
-func (m *mockRelay) defaultHandleRegisterValidator(w http.ResponseWriter, req *http.Request) {
+func (m *Relay) defaultHandleRegisterValidator(w http.ResponseWriter, req *http.Request) {
 	payload := []builderApiV1.SignedValidatorRegistration{}
-	if err := DecodeJSON(req.Body, &payload); err != nil {
+	if err := json.NewDecoder(req.Body).Decode(&payload); err != nil {
```

Comment:
This has differing functionality. This will allow unknown fields.

Reply:
yep, fixed